tests/integ/test_byo_estimator.py

```diff
@@ -136,6 +137,7 @@ def test_async_byo_estimator(sagemaker_session, region, cpu_instance_type, train
         predictor = model.deploy(1, cpu_instance_type, endpoint_name=endpoint_name)
         predictor.serializer = _FactorizationMachineSerializer()
         predictor.deserializer = sagemaker.deserializers.JSONDeserializer()
+        predictor.content_type = "application/json"
```

Comment:
In my next PR, I'm going to add serializer and deserializer parameters to `deploy`.